main.tf

```diff
@@ -0,0 +1,417 @@
+######
```

Comment:
[imo]
自分だったら以下のファイルに分割するかも🐱

- main.tf(vpcとvpc_flow_logに関するリソース)
- nat-instance.tf（NATinstanceリソース）
- route-table.tf（ルートテーブル）
- subnet.tf（サブネット）

少々極論だとは思うけど、オブジェクト指向だと1つの基準とされてるのが、クラス内のコードは100行、メソッドは5行までってのがある↓

https://www.jabba.cloud/20150821112745/

まあ分割するとかえってどこに何を置く？って考えるのが難しくなるし、「Sandi Metz’ Rules For Developers」はあくまでアプリケーションコードの話であってインフラストラクチャのコードの話じゃないし今のままでも全然OKな気もする:grin:

Reply:
@keitakn 同じく1つのファイルが長いかなと思ったけど、[Standard Module Structure](https://www.terraform.io/docs/modules/index.html#standard-module-structure
)に下記の通り記載されているから 1つの`main.tf`にしたよ！( complex module の定義が微妙だけど)
> For a simple module, this may be where all the resources are created. For a complex module, resource creation may be split into multiple files but any nested module calls should be in the main file.

あと、下記のmoduleをみても基本的に`main.tf`に記載されていて、Terraform Moduleとして公開するならこれに習うのがいいかと思ったよ🐱
https://github.com/terraform-aws-modules

プロジェクト固有のmoduleの場合は変更も多いと思うからコメントにもらったサイズを意識した方がいいかなと思った！

Reply:
@kobayashi-m42 なるほど🐱！方針了解👍
であれば今の形で問題ないと思う:grin: